microsoft-edge/extensions-chromium/publish/api/using-addons-api.md

```diff
@@ -94,24 +204,69 @@ https://login.microsoftonline.com/5c9eedce-81bc-42f3-8823-48ba6258b391/oauth2/v2
 }
 ```
 
-For more information, see [Get a token](/azure/active-directory/develop/v2-oauth2-client-creds-grant-flow#get-a-token) in _Microsoft identity platform and the OAuth 2.0 client credentials flow_.
+For more information, see [Get a token](/azure/active-directory/develop/v1.1-oauth2-client-creds-grant-flow#get-a-token) in _Microsoft identity platform and the OAuth 2.0 client credentials flow_.
+
+---
 
 
 <!-- ====================================================================== -->
 ## Using the API endpoints
 
-After you have an access token, you can use the Microsoft Edge Add-ons API.  This API exposes endpoints for getting a list of products, updating products, and publishing products.
+The Update REST API exposes endpoints for:
+* [Uploading a package to update an existing submission](#uploading-a-package-to-update-an-existing-submission)
+* [Checking the status of a package upload](#checking-the-status-of-a-package-upload)
+* [Publishing the submission](#publishing-the-submission)
+* [Checking the publishing status](#checking-the-publishing-status)
+
+These endpoints are introduced below.
+
+To use this Microsoft Edge Add-ons Update REST API, you must have the following, per above:
+
+##### [v1.1](#tab/v1-1)
+
+An API key and a Client ID.
 
-> [!NOTE]
-> There's no API for creating a new product or updating a product's metadata. For example, the description.  You must complete these tasks manually in Microsoft Partner Center.
+##### [v1](#tab/v1)
 
-The API is available at the endpoint `https://api.addons.microsoftedge.microsoft.com`
+An access token.
+
+---
+
+The API is available at the endpoint root `https://api.addons.microsoftedge.microsoft.com`.
+
+There aren't REST API endpoints for:
+* Creating a new product.
+* Updating a product's metadata, such as the description.
+
+To create a new product or update a product's metadata, you must use Microsoft Partner Center.
 
 
 <!-- ====================================================================== -->
 ## Uploading a package to update an existing submission
+<!-- endpoint 1: POST /submissions/draft/package -->
+
+Use this REST API endpoint to update the package for an add-on.  This API uploads a package to update an existing draft submission of an add-on product.
+
+See also [Upload a package to update an existing submission](./addons-api-reference.md#upload-a-package-to-update-an-existing-submission) in _REST API Reference for updating Microsoft Edge Add-ons_.
+
+
+<!-- ---------------------------------------------- -->
+#### Endpoint description
+
+##### [v1.1](#tab/v1-1)
+
+```REST
+Endpoint: /v1/products/$productID/submissions/draft/package
```

Comment:
```suggestion
Endpoint: /v1-1/products/$productID/submissions/draft/package
```
in v1.1 tabs in using .md, change /v1/ to /v1-1/ ?
@yvsubhash @bhuvanapriyap

Reply:
Ans: URL continues to be /v1/, that's why we changed from "v2" to "v1.1".  
url is same for v1.1 as for v1